api/libproject/devstart.rb

```diff
@@ -3221,4 +3221,104 @@ def send_email(cmd_name, *args)
     :invocation => SEND_EMAIL_CMD,
     :description => "Sends a system email.  Currently limited to egress emails.",
     :fn => ->(*args) {send_email(SEND_EMAIL_CMD, *args)}
+})
+
+def run_genomic_extraction(cmd_name, *args)
+  common = Common.new
+
+  op = WbOptionsParser.new(cmd_name, args)
+
+  op.add_typed_option(
+    '--project [project]',
+    String,
+    ->(opts, v) { opts.project = v },
+    'AoU environment GCP project full name. Used to pick MySQL instance & credentials.')
+  op.opts.project = TEST_PROJECT
+
+  op.add_typed_option(
+    '--namespace [workspace namespace]',
+    String,
+    ->(opts, v) { opts.namespace = v },
+    'The workspace namespace to run the extraction from.')
+
+  op.add_typed_option(
+    '--dataset_id [dataset id]',
+    String,
+    ->(opts, v) { opts.dataset_id = v },
+    'The dataset to record in the DB as associated with this extraction (arbitrary but must exist).')
+
+  op.add_typed_option(
+    '--person_ids ["id1, id2, id3"]',
+    String,
+    ->(opts, v) { opts.person_ids = v },
+    'The person IDs to be used in the extraction.')
+
+  op.add_typed_option(
+    '--legacy [true/false]',
+    String,
+    ->(opts, v) { opts.legacy = v },
+    'Use legacy (v7 and earlier) workflow (true) or v8+ workflow (false).')
+
+  op.add_typed_option(
+    '--filter_set [filter set]',
+    String,
+    ->(opts, v) { opts.filter_set = v },
+    'Filter set name.')
```

Comment:
what's this?

Reply:
It's one of the parameters to the workflow.  I don't know much about what the different options are here.